app/src/main/java/com/nextcloud/talk/controllers/SettingsController.java

```diff
@@ -144,6 +144,8 @@
     EmojiTextView displayNameTextView;
     @BindView(R.id.base_url_text)
     TextView baseUrlTextView;
+    @BindView(R.id.server_age_warning_text)
```

Comment:
Lint increase comes from this:
> Resource IDs will be non-final in Android Gradle Plugin version 5.0, avoid using them as annotation attributes

Can we ignore it for now?

Reply:
ignore it by using
@SuppressLint("NonConstantResourceId") @BindView(R.id.server_age_warning_text)

these lint warnings will be gone when we switched to view binding https://developer.android.com/topic/libraries/view-binding

Reply:
Nah, please raise the score in the ```lint-results.txt``` file so the issue gets reported but doesn't break the build. We will migrate to native view bindings after 12.0 :)

Reply:
otherwise raising the lint-score would most likely mean to often have another round of CI build..? But yeah, fine for me

Reply:
> otherwise raising the lint-score would most likely mean to often have another round of CI build..? But yeah, fine for me

Ignore would mean it is hidden in the reports, but might be fine, since compiler will terminate anyways as soon as we remove Butterknife dependency. You will always have a new round since changeign the results txt or the java/kotlin file is a commit/push ;)